tests/unit/ml_handlers/base_ml_test.py

```diff
@@ -0,0 +1,42 @@
+import os
+import time
+import pandas as pd
+
+from mindsdb_sql import parse_sql
+from ..executor_test_base import BaseExecutorTest
+
+
+class BaseMLTest(BaseExecutorTest):
```

Comment:
Please open an issue so that we eventually refactor all ML tests to use this class 👍

Reply:
Will do, @paxcema.